README.md

```diff
@@ -1 +1,151 @@
-# React_Capstone_Project
\ No newline at end of file
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 React Capstone <a name="about-project"></a>
+
+**React Capstone** was built to inform different kinds of cuisine.You can find many cuisines and recipe
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>HTML ,CSS ,JavaScript and React</summary>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Different kinds of cuisines**
+- **Detailed recipes**
+- **Navigating to detail**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
Hi @MayPyone,

I hope you're having a pleasant day! It appears that the video presentation, a requisite component of the project submission, is currently missing. The presentation is a valuable piece of your submission, allowing you to showcase the functionalities and features of your project and articulate aspects of your code that you find particularly intriguing or noteworthy. 🌟

### **Action Required: 🛠️**

1. **Record the Video Presentation:** Please record a video presenting the features of your project. The maximum length of the video is 5 minutes. Ensure that your presentation is clear, concise, and in professional English.
2. **Include Essential Elements:** Your video should comprise a succinct description of the project, its purpose, and a demonstration of all the features of your project. Additionally, please highlight a segment of the code that you find interesting or are proud of creating.
3. **Use Recommended Tools:** It's advised to use Loom for recording the video, but feel free to use any tool you are comfortable with as long as the quality of the video is maintained and it is easily accessible.

### **Suggestions: 🌿** [OPTIONAL]
- When recording, ensure that your voice is clear, and the video is free of unnecessary disruptions.
- Practice going over the points you want to discuss beforehand to ensure smooth delivery within the time constraint.

Once the video has been recorded, please include the link to it in your submission, and feel free to tag me or drop a comment in the PR thread so that I can review it promptly.

Looking forward to seeing your presentation and hearing about your experiences and learning while building this project! 🌟

Keep up the great work, and happy coding! 🚀

Best Regards,
Temitope